inc/block-patterns.php

```diff
@@ -12,11 +12,11 @@
  */
 function twentytwentytwo_register_block_patterns() {
 	$block_pattern_categories = array(
-		'twentytwentytwo-general' => array( 'label' => __( 'Twenty Twenty-Two General', 'twentytwentytwo' ) ),
-		'twentytwentytwo-footers' => array( 'label' => __( 'Twenty Twenty-Two Footers', 'twentytwentytwo' ) ),
-		'twentytwentytwo-headers' => array( 'label' => __( 'Twenty Twenty-Two Headers', 'twentytwentytwo' ) ),
-		'twentytwentytwo-query'   => array( 'label' => __( 'Twenty Twenty-Two Posts', 'twentytwentytwo' ) ),
-		'twentytwentytwo-pages'   => array( 'label' => __( 'Twenty Twenty-Two Pages', 'twentytwentytwo' ) ),
+		'featured' => array( 'label' => __( 'Featured', 'twentytwentytwo' ) ),
+		'footer' => array( 'label' => __( 'Footers', 'twentytwentytwo' ) ),
+		'header' => array( 'label' => __( 'Headers', 'twentytwentytwo' ) ),
+		'query'   => array( 'label' => __( 'Query', 'twentytwentytwo' ) ),
```

Comment:
I'm not a big fan of the "Query" name here, but it's used in Core right now so I think we should stick to it.